lib/lrama/grammar.rb

```diff
@@ -507,6 +508,21 @@ def fill_sym_to_rules
       end
     end
 
+    # @rbs () -> void
+    def validate_precedence_is_term!
```

Comment:
I think it might be better to name this method as  `#validate_no_precedence_for_nterm`.